### ISSUE TITLE
make get-deps returns a glide error

### ISSUE THREAD

**Issue by Stii** [statsg/statsg#1]
```bash
$ make get-deps
glide install
make: glide: No such file or directory
make: *** [get-deps] Error 1
```

**Comment by Stii** [comment]
Probably need to mention in the docs that glide is a pre requisite or install via the Makefile if not installed.

**Comment by ChristianWitts** [comment]
Thanks for that. I'll fix it up :+1:

**Comment by ChristianWitts** [comment]
Closed with #3
